src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClient.cs

```diff
@@ -402,6 +402,55 @@ private async Task<ResponseStreamHolder> CaptureLogsAsync(HttpMethod method, str
             throw await CreateUnexpectedStatusCodeExceptionAsync(responseBox.Value).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// GET /trace?pid={pid}&profile={profile}&durationSeconds={duration}
+        /// </summary>
+        public Task<ResponseStreamHolder> CaptureTraceAsync(int pid, TimeSpan duration, TraceProfile? profile, CancellationToken token)
+        {
+            return CaptureTraceAsync(GetProcessQuery(pid: pid), duration, profile, token);
+        }
+
+        private Task<ResponseStreamHolder> CaptureTraceAsync(string processQuery, TimeSpan duration, TraceProfile? profile, CancellationToken token)
+        {
+            return CaptureTraceAsync(
+                HttpMethod.Get,
+                CreateTraceUriString(processQuery, duration, profile),
+                content: null,
+                token);
+        }
+
+        private async Task<ResponseStreamHolder> CaptureTraceAsync(HttpMethod method, string uri, HttpContent content, CancellationToken token)
+        {
+            string contentType = ContentTypes.ApplicationOctetStream;
+
+            using HttpRequestMessage request = new(method, uri);
```

Comment:
I'm wondering if long term we could have more shared code between EgressTrace and CaptureTrace but I don't think we need to worry about that now.